src/utils/countSamples.ts

```diff
@@ -1,124 +1,110 @@
+import { getFormControlLabelUtilityClasses } from "@mui/material";
 import { Node, GISAIDRecord, HomeGeo } from "../d";
 import { getNodeAttr } from "./treeMethods";
 
 export type GeoLevels = "location" | "division" | "country";
 
-export const getNodeCounts = (
-  nodes: Node[],
-  geoLevel: GeoLevels,
+const matchesGeography = (
+  type: "gisaid" | "node",
   homeGeo: HomeGeo,
-  minYear: number,
-  minMonth: number,
-  maxYear: number,
-  maxMonth: number
+  geoLevel: string,
```

Comment:
The `geoLevel` here should use the `GeoLevels` type above to enforce usage. I see it used to use it before, so I'm not sure why it changed with this PR?

Reply:
ty! oversight